src/sys/kernel/random.rs

```diff
@@ -73,7 +73,7 @@ mod tests {
         // The kernel support section in the root lib.rs file says that we only aim to support >= 2.6 kernels,
         // so only test that case
         let poolsize = poolsize().unwrap();
-        assert!(poolsize == 4096)
+        assert_eq!(poolsize, 4096)
```

Comment:
This will conflict with #205.  That's OK, just FYI

Reply:
I remove it too.